core/src/components/item/item.ionic.scss

```
@@ -112,3 +111,13 @@ slot[name="end"]::slotted(*) {
 :host(.item-lines-none) {
   --inner-border-width: #{globals.$ion-border-size-0};
 }
+
+// Inside a Select Modal
```

Let's follow the structure.

```suggestion
// Item in Select Modal
// --------------------------------------------------
```